### PR TITLE
Autocompletion for fields and expressions asking a function parameter name

### PR DESCRIPTION
**Pull request by D8H**
I used an expression field. I'm not sure this is really necessary, but it let the user knows when quotes are missing.

ParameterError

ParameterAutocompletion

### CODE REVIEW

**Review by 4ian — APPROVED** [review]
Looks good after the comment is addressed (+autoformatting ;))

**Comment by D8H** [comment]
> (+autoformatting ;))

I did it on gdjs out of habit 😅

**Review by 4ian — APPROVED** [review]

**Comment by 4ian** [comment]
Awesome!

File: newIDE/app/src/ExpressionAutocompletion/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ import { getVisibleParameterTypes } from '../EventsSheet/ParameterFields/Generic
 import { getParameterChoices } from '../EventsSheet/ParameterFields/ParameterMetadataTools';
 import getObjectByName from '../Utils/GetObjectByName';
 import { getAllPointNames } from '../ObjectEditor/Editors/SpriteEditor/Utils/SpriteObjectHelper';
-import { mapFor } from '../Utils/MapFor';
+import { enumerateParametersUsableInExpressions } from '../EventsSheet/ParameterFields/EnumerateFunctionParameters';
```
Comment (4ian) [inline newIDE/app/src/ExpressionAutocompletion/index.js:25]: Great naming for this file :)

**Pull request merged**